components/Footer/Footer.tsx

```diff
@@ -0,0 +1,9 @@
+export default function Footer() {
+  return (
+    <footer>
+      <div className="container">
+        <p>all works are copyright Oly Su 2022</p>
```

Comment:
Note: we can use JavaScript to get current year. Happy to revisit later when we work on the styling epic.